frameworks/PHP/codeigniter/install.sh

```diff
@@ -1,13 +1,12 @@
 #!/bin/bash
 
 export PHP_HOME=${IROOT}/php-5.5.17
-
-export PHP_FPM=$PHP_HOME/sbin/php-fpm
-
+export COMPOSER_HOME=${IROOT}/php-composer
+export PHP_FPM=${PHP_HOME}/sbin/php-fpm
 export NGINX_HOME=${IROOT}/nginx
 
 fw_depends php nginx composer
 
-${PHP_HOME}/bin/php $IROOT/composer.phar install \
+${PHP_HOME}/bin/php ${COMPOSER_HOME}/composer.phar install \
```

Comment:
Yes - composer.phar is now at $IROOT/php-composer/composer.phar because of the change made in this PR at toolset/setup/linux/systools/composer.sh. (I just went through and double checked that all of the references to COMPOSER_HOME instead of IROOT)